### DISCUSSION TITLE
Can there be a "Save and edit entry/entries" button after editing the field layout of a Section?

### DISCUSSION THREAD

**Discussion by proimage** [craftcms/cms#8875] [Ideas]
### Description
Often when I'm editing the field layout of a Section—especially a Single—I find I immediately want to go and edit the actual entry with its new fields. Similarly to how editing entries allows a "Save and continue editing" option, it would be handy for the Edit Entry Type page's Save button to have an alternate save-y option for "Save and edit entry".

**Comment by mmikkel** [comment]
This is kind of related to #6133 – especially @wbrowar's [comment](https://github.com/craftcms/cms/issues/6133#issuecomment-664394892) on that issue, which mentions the "edit source" button that the [CP Field Inspect plugin](https://plugins.craftcms.com/cp-field-inspect) adds to element edit pages. For entries specifically, the plugin adds an "Edit Entry Type" button to the details sidebar, which redirects the user to the settings page for the relevant entry type, and back to the entry when the entry type settings form is saved.

I'd love it if something similar was added to core - the number of mouse clicks and pageloads it saves really adds up, and it helps a lot in speeding things up during the content modelling/development phase for a build.

![image](https://user-images.githubusercontent.com/298510/88777103-2339c480-d187-11ea-9ea7-199cfe5907dc.png)

**Comment by brandonkelly** [comment]
Yeah agreed. We’re going to look into reducing clicks in 3.6.
